web/src/main.js

```diff
@@ -34,7 +36,9 @@ ReactDOM.render(
     {/* the route history outside of React components anymore */}
     <Router history={history}>
       <GlobalStyle />
-      <Main />
+      <ThemeProvider theme={themes.colorblindDark}>
```

Comment:
You'll need to provide the right theme to `ThemeProvider` which can really be any of the 4, based on the values of `brightModeEnabled` and `colorBlindModeEnabled` - fortunately there is already a `useTheme` hook which you could reuse here:

```suggestion
      <ThemeProvider theme={useTheme()}>
```

Since the hook depends on the Redux store provider, you might get an error trying to use it here, so it might be easiest if you actually moved `<ThemeProvider>` back to `src/layout/main.js` (sorry for misleading you before :grimacing:)